src/components/attendee/IntersectingButton/IntersectingButton.tsx

```diff
@@ -1,21 +1,12 @@
 import { ButtonHTMLAttributes, RefObject, useEffect, useRef } from "react";
 import { useIntersection } from "react-use";
-import { Button } from "components/attendee/Button";
-
-// Button And Border variant types are the same to allow variant mixing.
-// But we might have different variants for either button or border in the future
-type ButtonVariant = "primary" | "alternative";
-type BorderVariant = ButtonVariant;
+import { Button, ButtonProps } from "components/attendee/Button";
 
 interface IntersectingButtonProps
-  extends ButtonHTMLAttributes<HTMLButtonElement> {
-  onClick?: () => void;
-  variant?: ButtonVariant;
-  transparent?: boolean;
-  border?: BorderVariant;
-  className?: string;
-  forwardRef?: RefObject<HTMLButtonElement>;
+  extends ButtonHTMLAttributes<HTMLButtonElement>,
+    ButtonProps {
   updateIntersected: (el: RefObject<HTMLButtonElement>) => void;
+  onClick?: () => void;
```

Comment:
I don't think this one is needed, by deleting `onClick` from 3 places it can just be passed through as part of `rest` props

Reply:
this caused another TS error saying that `onClick` types differ from `htmlbutton` one
not sure why, leaving it as it is for now